app/posts/manage-teacher-training-applications/2021-01-06-closing-an-application.md

```diff
@@ -0,0 +1,151 @@
+
+- Replaced the words ‘reject application’ and ‘withdraw offer’ to ‘Close application’ so that users do not get hung up on terminology
+- Replace ‘Declined’, ‘Rejected’, ‘Offer withdrawn’ and ‘Application withdrawn’ statuses with ‘Closed’
+- Always let users close an application (unless it’s already been closed) so that there’s a consistent way to close an application regardless of the status
+- Kept all questions on the first screen for closing an application as it’s more work to split them up for this round of research
+- We put questions that are not a reflection of the candidate toward the bottom of the form
+- Once an offer has been made, irrelevant questions like safeguarding and performance at interview are removed
+- Let providers give additional feedback when the course is full because if the candidate was poor, then the course being full is not the main reason for closing the application
+- Let users navigate easily to change the status of a condition if they choose ‘Close application’ when in the ‘Accepted’ state
+- Kept the ‘performance at interview’ question in the ‘Received’ state because a provider may not use the interview feature
+
+## Giving a choice back to candidates under certain circumstances
+
+We think that candidates should be given a choice back if their application is closed through no fault of their own. For example, when the course is full.
+
+## Other options we discarded for now
+
+We originally put ‘Close application’ in a sidebar on the application page.
+
+But from previous reseach and on the basis that we currently have ‘Withdraw offer’ on the offer page, we think that users will need to close an application from the offer page (and potentially other pages).
+
+We also tried to only show ‘Close application’ under the candidate’s name when in the offered, accepted and conditions met states. This seemed sensible on the basis that the prompts let users close the application - like when making a decision.
+
+But that meant an application couldn’t be closed when in the deferred state. So we decided to put ‘Close application’ just under the name at all times.
+
+## What we want to know from research
+
+- Do users understand what ‘Close application’ means?
+- Can users figure out how to close an application when their mental model is to withdraw an application?
+- Do some users go to close an application when they want to mark a condition as not met?
+- Do users need to distinguish between applications that were closed for different reasons on the application list?
+- Do users mistake ‘Close application’ for going back to the application list?
+
+## Other things to explore later
+
+- Maybe the reason about qualifications overlaps with why a condition was not met. For example, they could not provide evidence of a qualification
+- The ‘offered a place elsewhere’ reason could a dedicated flow once UCAS is gone because Manage knows that a place was offered elsewhere. Peraps it could even be automated.
```

Comment:
```suggestion
- The ‘offered a place elsewhere’ reason could a dedicated flow once UCAS is gone because Manage knows that a place was offered elsewhere. Perhaps it could even be automated.
```